### ISSUE TITLE
Add space-around justification option to Buttons Block

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#42536]
## What problem does this address?
The buttons block provides four alignment options which does not include all the potential values for `justify-content` in a flex container. I find that the `space-around` value in particular can be a really nice way of centering and filling available space, and so don't know why it's not available.

## What is your proposed solution?
Add support for `justify-content: space-around` to the Buttons block. Consider also adding `justify-content: space-evenly` for nearly full coverage of [all possible values](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content).

**Comment by jasmussen** [comment]
Possibly related: #44467

**Comment by jasmussen** [comment]
Found some time for an idea. How about this?

<img width="270" height="129" alt="Image" src="https://github.com/user-attachments/assets/662fec7b-f633-4024-b76a-a8056793a563" />

The Layout panel's "Justification" control becomes a select, so it can legibly fit 5 options. The select opens this:

<img width="292" height="241" alt="Image" src="https://github.com/user-attachments/assets/436e8e7f-72d4-42a8-b9f5-8f6ef2e1236a" />

The latter—radios and icons—is possible with [this component](https://wordpress.github.io/gutenberg/?path=/story/design-system-components-menu--radio-items), the question is whether "Select" as the opener is the right one. I defer to @WordPress/gutenberg-components here.

**Comment by jasmussen** [comment]
Noting that the above adds "Space evenly", if we would like to also add "Space around", we can do that too insofar as the dropdown should fit it. Let me know, happy to draw an additional icon.

**Comment by jasmussen** [comment]
Also, not quite feeling the following, it's not clear this one will be userfriendly in practice, so sharing it mostly for completeness:

<img width="270" height="129" alt="Image" src="https://github.com/user-attachments/assets/ca5a9562-28c3-41d8-bdae-708902e8fa45" />

**Comment by ciampo** [comment]
I guess Select is my preferred option, but choosing Menu could also be an arguable choice.

@mirka the new `@wordpress/ui` select components would support custom icons, right?

**Comment by fcoveram** [comment]
I'm drawn to keep using the existing interactions (icon dropdown menu or toggle group) rather than using a new component control.

**Comment by jasmussen** [comment]
You mean [this one then](https://github.com/WordPress/gutenberg/issues/42536#issuecomment-5368859425), yes?

**Comment by fcoveram** [comment]
> You mean https://github.com/WordPress/gutenberg/issues/42536#issuecomment-5368859425, yes?

Yes. Since the dropdown menu is triggered from the toolbar, I find odd to interact with a label-based control.
